app/tests/unit/containers/Incentives/IncentiveCalculatorContainer.test.tsx

```diff
@@ -27,6 +57,15 @@ describe('IncentiveCalculator', () => {
     expect(r).toMatchSnapshot();
   });
 
+  it('includes an aggregate row with multiple products', async () => {
+    const r = shallow(
+      <IncentiveCalculator
+        applicationRevision={applicationRevisionMultiProduct}
+      />
+    );
+    expect(r).toMatchSnapshot();
+  });
+
```

Comment:
There are a bunch of `todo` tests at the bottom of this file, which are all outdated. Let's remove them.